index.js

```diff
@@ -115,18 +114,31 @@ var wcI18nV2 = function(file, encoding, callback) {
     }).filter(function(jsStringObj) {
       return /\bWCI18n\b/.test(jsStringObj.text);
     }).map(function(jsStringObj) {
-      return {
-        script: jsStringObj.script,
-        tree: ast(jsStringObj.text)
-      };
-    }).filter(function(treeObj) {
-      return treeObj.tree.callExpression('WCI18n').length > 0;
-    }).map(function(treeObj) {
-      var componentName = treeObj.tree.callExpression('Polymer').arguments.at(0).key('is').value();
+      var componentName, callNode;
+
+      var output = falafel(jsStringObj.text, function (node) {
+        // console.log('node', require('util').inspect(node, { depth: null }));
+
+        // check for Polymer 1.x constructor: Polymer({is:'my-element'})
+        if (node.type === 'Property' && node.key.name === 'is') {
+          componentName = node.value.value;
```

Comment:
Should we handle exceptions that may be thrown because a property is undefined? i.e  `node` or `node.value` is undefined?   Same below with `node.value.body.body[0].argument.value`